### PR TITLE
tls 1.3 updates; fix typo

### PR DESCRIPTION
**Pull request by kelseiv**
Closes https://github.com/influxdata/docs-v2/issues/1465

### CODE REVIEW

File: content/enterprise_influxdb/v1.8/administration/config-data-nodes.md [hunk 1]
```diff
@@ -1222,32 +1229,21 @@ ciphers = [ "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305",
...
-
-    The maximum number of shards that a single data node will copy or repair in parallel.
-
-    Environment variable: `INFLUXDB_ANTI_ENTROPY_MAX_FETCH`
-
-    > Having `max-fetch=10` with higher numbers of shards (100+) can add significant overhead to running nodes.
-    > The more shards you have, the lower this should be set.
-    > If AE is left enabled while lowering your `max-fetch`, you will initially see
-    > higher CPU load as new shard digest files are created.
-    > The added load will drop off after shard digests are completed for existing shards. -->
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.8/administration/config-data-nodes.md:1253]: @pierwill, please verify these are same TLS updates you made in 1.9. 

Noticed a couple typos in this doc that I fixed in this PR(AE setting commented out in the TLS section and fixed header), so we'll want to pull those over into the 1.9 branch to resolve merge conflicts.

File: content/enterprise_influxdb/v1.8/administration/config-data-nodes.md [hunk 2]
```diff
@@ -1222,32 +1229,21 @@ ciphers = [ "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305",
...
-min-version = "tls1.2"
+min-version = "tls1.3"
 
-max-version = "tls1.2"
+max-version = "tls1.3"
 
 ```
 
-#### `min-version = "tls1.2"`
+#### `min-version = "tls1.3"`
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.8/administration/config-data-nodes.md:1238]: @pierwill, please verify these are same TLS updates you made in 1.9.

File: content/enterprise_influxdb/v1.8/administration/config-data-nodes.md [hunk 3]
```diff
@@ -1222,32 +1229,21 @@ ciphers = [ "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305",
...
 
 Environment variable: `INFLUXDB_TLS_MIN_VERSION`
 
-#### `max-version = "tls1.2"`
+#### `max-version = "tls1.3"`
 
-The maximum version of the TLS protocol that will be negotiated. Valid values include: `tls1.0`, `tls1.1`, and `tls1.2`. If not specified, `max-version` is the maximum TLS version specified in the [Go `crypto/tls` package](https://golang.org/pkg/crypto/tls/#pkg-constants). In this example, `tls1.2` specifies the maximum version as TLS 1.2, which is consistent with the behavior of previous InfluxDB releases.
+The maximum version of the TLS protocol that will be negotiated. Valid values include: `tls1.0`, `tls1.1`, `tls1.2` and `tls1.3`. If not specified, `max-version` is the maximum TLS version specified in the [Go `crypto/tls` package](https://golang.org/pkg/crypto/tls/#pkg-constants). In this example, `tls1.3` specifies the maximum version as TLS 1.3, which is consistent with the behavior of previous InfluxDB releases.
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.8/administration/config-data-nodes.md:1246]:
```suggestion
The maximum version of the TLS protocol that will be negotiated. Valid values include: `tls1.0`, `tls1.1`, `tls1.2` and `tls1.3`. If not specified, `max-version` is the maximum TLS version specified in the [Go `crypto/tls` package](https://golang.org/pkg/crypto/tls/#pkg-constants). In this example, `tls1.3` specifies the maximum version as TLS 1.3.
```

File: content/enterprise_influxdb/v1.8/administration/config-data-nodes.md [hunk 4]
```diff
@@ -1222,32 +1229,21 @@ ciphers = [ "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305",
...
-max-version = "tls1.2"
+max-version = "tls1.3"
 
 ```
 
-#### `min-version = "tls1.2"`
+#### `min-version = "tls1.3"`
 
-Minimum version of the TLS protocol that will be negotiated. Valid values include: `tls1.0`, `tls1.1`, and `tls1.2`. If not specified, `min-version` is the minimum TLS version specified in the [Go `crypto/tls` package](https://golang.org/pkg/crypto/tls/#pkg-constants). In this example, `tls1.0` specifies the minimum version as TLS 1.0, which is consistent with the behavior of previous InfluxDB releases.
+Minimum version of the TLS protocol that will be negotiated. Valid values include: `tls1.0`, `tls1.1`, `tls1.2` and `tls1.3`. If not specified, `min-version` is the minimum TLS version specified in the [Go `crypto/tls` package](https://golang.org/pkg/crypto/tls/#pkg-constants). In this example, `tls1.0` specifies the minimum version as TLS 1.0, which is consistent with the behavior of previous InfluxDB releases.
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.8/administration/config-data-nodes.md:1240]:
```suggestion
Minimum version of the TLS protocol that will be negotiated. Valid values include: `tls1.0`, `tls1.1`, `tls1.2` and `tls1.3`. If not specified, `min-version` is the minimum TLS version specified in the [Go `crypto/tls` package](https://golang.org/pkg/crypto/tls/#pkg-constants). In this example, `tls1.3` specifies the minimum version as TLS 1.3.
```

File: content/enterprise_influxdb/v1.8/administration/config-data-nodes.md (hunk 2 above)
Comment (pierwill) [reply content/enterprise_influxdb/v1.8/administration/config-data-nodes.md:1238]: Looks good!

**Review by pierwill — APPROVED** [review]
Just a few changes. In 1.9, I removed the parts about "which is consistent with the behavior of previous InfluxDB releases".

File: content/enterprise_influxdb/v1.8/administration/config-data-nodes.md (hunk 1 above)
Comment (pierwill) [reply content/enterprise_influxdb/v1.8/administration/config-data-nodes.md:1253]: created https://github.com/influxdata/docs-v2/issues/2603

**Comment by pierwill** [comment]
Thanks, @kelseiv!

**Pull request merged**